main.go

```diff
@@ -263,30 +272,110 @@ func (svr *bridge) handleCall(w http.ResponseWriter, r *http.Request) {
 			}
 
 			if val, ok := alert.Annotations[*svr.titleAnnotation]; ok {
-				title += val
+				titleTemplate, err := template.New("title").Parse(val)
```

Comment:
A thought that may or may not be realistic... what if the text intentionally includes `{{` in it, but is not intended to be used as a template? Is there a way to escape this with templating?